src/models/network.ts

```diff
@@ -0,0 +1,54 @@
+class Network {
+  public title: string;
+  public totalAtom: string;
```

Comment:
I still prefer you change the variable names to a generic one.
```suggestion
  public totalTokens: string;
```